src/Applications/GEOSdas_App/fvsetup

```diff
@@ -579,21 +579,22 @@ $ENV{"PATH"} = $FindBin::Bin .":$ENV{PATH}";
   die "Failed to fix GOCART levels-referencing $!" if ( $rc );
 
 # Call LDAS setup script.
-#  Provide LDAS config input files in $ldassetup/: det_exeinp.txt, det_batinp.txt. 
-#  For Hyb-4dEnVar, do same for ens_exeinp.txt, ens_batinp.txt. 
+#  Provide LDAS config input files in $ldassetup/: ldas_exeinp.txt, ldas_batinp.txt. 
 #  See GEOSldas/README.md for generating sample config input files.
 # ------------------------------------------------------------------------
     print "\n checkprintout  ldas_ana == 1 ($ldas_ana)\n";
     if ($ldas_ana == 1 ) {
-        $cmd = "$fvbin/ldas_setup setup $fvhome/lana --account $gid --nymdb $nymdb --nhmsb $nhmsb $ldassetup/det_exeinp.txt $ldassetup/det_batinp.txt";
+        $cmd = "$fvbin/ldas_setup setup $fvhome/lana --account $gid --nymdb $nymdb --nhmsb $nhmsb --ladas_cpl 1 --agcm_res $res --rstdir $ldasics $ldassetup/ldas_exeinp.txt $ldassetup/ldas_batinp.txt";
```

Comment:
Should we call ldas_setup with the "-v" (--verbose) option?  Not sure if this would/should be captured in the ADAS setup log, assuming such a log exists.

Reply:
It is cleaner to put all non-positional params after positional ones